.golangci.yml

```diff
@@ -1,6 +1,6 @@
 linters-settings:
   depguard:
-    list-type: blacklist
+    list-type: denylist
```

Comment:
My IDE was showing an error for this change based on the [JSON schema here](https://json.schemastore.org/golangci-lint.json). So not sure if this is OK to change or not.